app/components/getChordprinter/Testimonial.tsx

```diff
@@ -0,0 +1,141 @@
+'use client';
+
+import Image from 'next/image';
+import React, { useRef } from 'react';
+import { Autoplay, Pagination } from 'swiper/modules';
+import type { SwiperRef } from 'swiper/react';
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+import styles from './Testimonial.module.css';
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
+
+type TestimonialArray = {
+  image: string;
+  reference: string;
+  author: string;
+};
+
+const Testimonial = () => {
+  const sliderRef = useRef<SwiperRef>(null);
+
+  const testimonialArray: TestimonialArray[] = [
```

Comment:
tady to pole nezavisi na zadnym hooku ci propsu, tak se vyplati ho deklarovat mimo tu komponentu. Kdyz se totiz ta komponenta prerenderuje, tak se to pole deklaruje znovu a musi se vytvorit novy policko v pameti pro nej, to stary vyhodit garbage collectorem atd. Ono to v tomto pripade nevadi, pac je to pole malicky a navic se ta komponenta neprerendruje, ale je dobry vyvarovat se temto antipaternam. Kdybys takto deklaroval funkci treba, tak ti to eslint chyti, ale dovnitr objektu se asi nediva